controllers/courses.js

```diff
@@ -620,99 +622,122 @@ router.get('/:courseId/', (req, res, next) => {
 	];
 
 	// ########################### start requests to new Editor #########################
+	let editorBackendIsAlive = true;
 	if (EDITOR_URL) {
-		promises.push(apiEditor(req).get(`course/${req.params.courseId}/lessons`));
+		promises.push(apiEditor(req)
+			.get(`course/${req.params.courseId}/lessons`)
+			.catch((err) => {
+				logger.warn('Can not fetch new editor lessons.', err);
```

Comment:
this is an error, isn't it?
```suggestion
				logger.error('Can not fetch new editor lessons.', err);
```